### PR TITLE
Handle more msgtypes smartly

### PR DESCRIPTION
**Pull request by tadzik**

### CODE REVIEW

File: src/bridge/mod.rs [hunk 1]
```diff
@@ -186,12 +186,43 @@ impl<IS: Io> Bridge<IS> {
...
+                        match url {
+                            Some(url) => self.irc_conn.send_message(&channel, sender_nick,
+                                                                    self.matrix_client.media_url(&url).as_str()),
+                            None      => warn!(self.ctx.logger, "Media message has no url"; "room" => room_id,
+                                                                                            "message" => format!("{:?}", ev))
+                        }
+                    },
+                    _ => {
+                        warn!(self.ctx.logger, "Unknown msgtype"; "room" => room_id, "msgtype" => msgtype);
+                        self.irc_conn.send_message(&channel, sender_nick, body);
```
Comment (erikjohnston) [inline src/bridge/mod.rs:224]: Generally if a `m.room.message` has a `body` we should probably display it to the user, as thats kinda the point of `body`

Comment (tadzik) [reply src/bridge/mod.rs:224]: Doesn't that line od exactly that?

Comment (erikjohnston) [reply src/bridge/mod.rs:224]: Err, yes. Yes it does.

*goes and finds some coffee*

**Review by erikjohnston — APPROVED** [review]
LGTM

**Comment by madduck** [comment]
@tadzik I now see a lot of those in my logs:

    WARN Unknown msgtype, room: !AybwxrssewHPXtIRIN:madduck.net, msgtype: m.notice

**Comment by tadzik** [comment]
Awesome, now we now what to add ;) I guess they should've been on the debug loglevel though.

**Pull request merged**